rocks-lib/src/package/mod.rs

```diff
@@ -124,6 +140,15 @@ impl From<RemotePackage> for PackageReq {
     }
 }
 
+#[cfg(feature = "lua")]
+impl mlua::UserData for PackageReq {
+    fn add_methods<M: mlua::UserDataMethods<Self>>(methods: &mut M) {
+        methods.add_meta_method("__eq", |_, this, package: RemotePackage| {
+            Ok(this.matches(&package))
+        });
+    }
+}
```

Comment:
question: Does `__eq` make sense here?
I'd lean towards a `matches` method instead.

Reply:
Good call - brain was on autopilot mode, I thought this was `RemotePackage:matches(&RemotePackage)`, but this is a PackageReq